### PR TITLE
add delete button and corresponding backbone destroy functionality

### PR DESCRIPTION
**Pull request by majornorth**

### CODE REVIEW

**Comment by majornorth** [comment]
@ezarowny +1? :+1:

File: utils/static/utils/scripts/app/views.js [hunk 1]
```diff
@@ -20,6 +22,10 @@ define(['jquery', 'underscore', 'backbone', 'marionette', 'app/models'], functio
         setDoneState: function() {
             var isDone = this.ui.itemDone.prop("checked");
             this.model.setDone(isDone);
+        },
+
+        deleteMofo: function() {
```
Comment (ezarowny) [inline utils/static/utils/scripts/app/views.js]: deleteTodo perhaps?

File: utils/templates/utils/index.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
     <script type="text/html" id="list-item">
         <input class="item-done" type="checkbox"></input>
-        <%= title %>
+        <%= title %><button class="delete">Delete!</button>
```
Comment (ezarowny) [inline utils/templates/utils/index.html]: Move button onto next line for readability's sake?

**Comment by majornorth** [comment]
@ezarowny updated based on your feedback. Mergeruskie!

**Comment by ezarowny** [comment]
Squash it and we're good to go!

**Pull request merged**